src/materials/nodes/manager/NodeMaterialObserver.js

```diff
@@ -182,10 +185,29 @@ class NodeMaterialObserver {
 
 				}
 
+			} else if ( value.isTexture === true ) {
+
+				if ( value.uuid !== mtlValue.uuid ) {
+
+					materialData[ property ] = value.clone();
```

Comment:
Could we add a object `{ version, uuid }` instead of cloning the texture?

Reply:
Will do, a bit lighter weight now we are special casing textures anyway.

Reply:
actually can use id rather than uuid now we are no longer cloning.